README.md

```diff
@@ -100,6 +100,23 @@ npm run build
 
 This command will build all packages in the monorepo in a correct order.
 
+### Running Tests
```

Comment:
I am not convinced this is the best place for network/package specific explanation on how to run package tests. I'd propose we move it to the package README (even though it is executed from toplevel).

Also the solana test you added is integration test, the average joe won't care about that and in general that should not be run as a "default" run because it's more likely to be prone to network & chain issues.

I would create a special test group smth like `test:integration` or `test:solana-integration` (I haven't give much thought to what it should look like, but for sure it should be separate flow)

Reply:
And I would do it as such that if the TEST_SOLANA_MNEMONIC is missing the integration test code outputs the warning and skips test. I think that is convinient as if, say we have integration tests for each package and I (the user) just run a `npm run test:integration` that triggers 10 tests, one per each pacakge.

Then I only have to set one env variable TEST_SOLANA_MNEMONIC and the rest will be skipped

Reply:
sounds good, on it